### DISCUSSION TITLE
With the programming cable connected, the display shows that the radio is charging, but also 100%

### DISCUSSION THREAD

**Discussion by OE1MWW** [egzumer/uv-k5-firmware-custom#73] [General]
Hi all,

power on / cable connected to the UV-5R+ / USB connected to the PC (sequence is important)
shows the text 'Charge 8.33V 100%' in the middle of display, on top left besides the battery symbol 
is a plug symmbol and further left is written 100% too.

But this seems to be a bug, it does not charge, battery is still dicharged by the power on state.

Is this a 'bug' or a 'feature' ?  ;-)

73's de OE1MWW
Wolfgang 

P.S. the 1o11_fagci_spectrum_v0.15_packed.bin is otherwise *perfect* - Thanks for that!

**Comment by rakenrowl7** [comment]
Hello,

May I ask what kind of cable you have? I measured my cable which is a CH340 'baofeng' cable and the RxD to ground outputs a voltage of 5v, which I presume to be taken by radio as current in, which triggers the 'usb charge mode' on the radio. I presume the cable does not actually have the current output to even charge the radio, so this may be a hardware bug on QS.

**Reply by OE1MWW** [reply]
@rakenrowl7 Two of my cables (original 'BAOFENG' and one unlabeled) are of CH340 type and there is no difference in the behavior described. See comment of egzumer in next post

**Comment by egzumer** [comment]
Yes this is hardware related. 5V UART cables interfere with voltage readings on the MCU.

**Comment by ErikS-web** [comment]
Title changed to: With the programming cable connected, the display shows that the radio is charging, but also 100%

Original title: cable connected / 1o11_fagci_spectrum_v0.15_packed.bin #73
